src/transformStateWithClones.js

```diff
@@ -6,8 +6,39 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  const currentState = { ...state };
```

Comment:
The name `currentState` for the object copy is appropriate and follows good naming conventions.